docs/src/App/App.tsx

```diff
@@ -59,122 +59,126 @@ export default withRouter(
                   <Logo />
                 </Link>
 
-                <button
-                  className={classnames({
-                    [styles.menuButton]: true,
-                    [styles.menuButton__isHidden]: isHome,
-                    [styles.menuButton__isOpen]: isMenuOpen && !isHome
-                  })}
-                  onClick={this.toggleMenu}
-                  aria-label={isMenuOpen ? 'Close menu' : 'Open menu'}
-                >
-                  <div
-                    className={classnames(
-                      styles.menuButton__bar,
-                      styles.menuButton__bar1
-                    )}
-                  />
-                  <div
-                    className={classnames(
-                      styles.menuButton__bar,
-                      styles.menuButton__bar2
-                    )}
-                  />
-                  <div
-                    className={classnames(
-                      styles.menuButton__bar,
-                      styles.menuButton__bar3
-                    )}
-                  />
-                </button>
+                <Hidden print>
```

Comment:
Dogfooding straight away 👍 🐶 🥘

Reply:
Yeah, trying to make this more of a habit.